src/components/ConfirmButton/ConfirmButton.tsx

```diff
@@ -44,16 +44,24 @@ function ConfirmButton({
             >
                 {children}
             </Button>
-            <Modal open={showConfirm && !confirmation.hide} onClose={closeModal} aria-labelledby={id}>
-                <Modal.Content className={styles.confirmationModalContent}>
-                    <Heading id={id} spacing level="2" size="medium" className={styles.confirmationModalHeader}>
-                        {confirmation.title}
-                    </Heading>
+            <Modal
+                header={{
+                    heading: confirmation.title,
+                    size: 'medium',
+                }}
+                open={showConfirm && !confirmation.hide}
+                onClose={closeModal}
+                aria-labelledby={id}
+                className={styles.confirmationModalContent}
+            >
+                <Modal.Body>
                     {confirmation.body.map((paragraph) => (
                         <BodyShort key={paragraph} spacing>
                             {paragraph}
                         </BodyShort>
                     ))}
+                </Modal.Body>
+                <Modal.Footer>
                     {confirmation.feedback}
```

Comment:
har du testet hvordan dette ser ut når feedabck ligger i footer? TrooooooooooooOOOOOr den burde ligge i body. Footer har veldig spesifikk CSS for plassering av 1, 2 eller 3 buttons. :stuck_out_tongue: